README.md

```diff
@@ -9,3 +9,15 @@ This repository includes the following CI images for RAPIDS:
 ## `latest` tag
 
 The `latest` image tags are controlled by the values in `latest.yaml`.
+
+## Building the dockerfiles locally
+To easily build the dockerfiles locally, you may use the following snippets:
+```sh
```

Comment:
```suggestion
## Building the dockerfiles locally

To easily build the dockerfiles locally, you may use the following snippets:

```sh
```

Markdown formatters generally put newlines between these elements. Let's add the newlines since we'll be doing one last round of edits.